scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -215,7 +215,12 @@ abstract class LocalScanner(name: String, config: ScannerConfiguration) : Scanne
                             "and $scannerDetails, not scanning the package again $packageIndex."
                 }
 
-                storedResults
+                // Due to the bug (fixed in the preceeding commit) the scan results for packages were not properly
```

Comment:
I believe code comments should not refer to Git commit history in this way, as such references are bound to break. I'd just say "Due to a temporary bug that has been fixed by now ..."

Reply:
Commit message: s/non applicable/non-applicable/

Reply:
Done.